bots/push-rewrite

```diff
@@ -54,6 +62,13 @@ def main():
     api = github.GitHub(repo=opts.repo)
     old_statuses = api.statuses(remote)
 
+    pull = find_pr_with_sha(api, remote)
+    labels = labels_of_pull(pull)
+    tests_enabled = "no-test" in labels or "[no-test]" in pull["title"]
+    # add no-test label to prevent tests bring triggered
+    if tests_enabled:
+        label(pull, labels + ["no-test"])
```

Comment:
This should be `tests_disabled`. However, why is that necessary? If the test already  has "no-test", then there's no need to do it again.

It seems to me that the condition for `tests_enabled` should be negated, so that this labels/unlabels the PR if `no-test` is *not* present?

Reply:
oh you're right, must've tested it then without removing the label. Fixing this.